index.bs

```diff
@@ -5290,17 +5290,15 @@ defined in this specification are applicable to [=typedefs=].
 
 In a given implementation of a set of [=IDL fragments=],
 an object can be described as being a <dfn id="dfn-platform-object" export>platform object</dfn>, a
-<dfn id="dfn-user-object" export>user object</dfn>, or neither.  There are two kinds of
-object that are considered to be platform objects:
+<dfn id="dfn-user-object" export>user object</dfn>, or neither.
 
-*   objects that implement a non-[=callback interface|callback=] [=interface=];
-*   objects representing an IDL {{DOMException}}.
+Platform objects are objects that implement a non-[=callback interface|callback=] [=interface=].
```

Comment:
While at it, I'd move the `<dfn>` for platform objects here, and similarly the `<dfn>` for user objects to "User objects are those that authors would create..." Then link to these comments from "an object can be described as being a platform object, a user object, or neither."